engines/bastion_katello/test/products/details/product-repositories.controller.test.js

```diff
@@ -46,12 +46,15 @@ describe('Controller: ProductRepositoriesController', function() {
             return {'$promise': $q.defer().promise};
         };
 
+        DownloadPolicy = $injector.get('MockResource').$new();
```

Comment:
I wouldn't use MockResource here but do the same as what you did here -- https://github.com/Katello/katello/pull/5758/files#diff-75098ed0639cce525980b3d7df39692fR11